src/main/java/picard/sam/markduplicates/UmiGraph.java

```diff
@@ -189,7 +203,11 @@ public UmiGraph(DuplicateSet set, String umiTag, String assignedUmiTag, boolean
         return duplicateSetList;
     }
 
-    // Create a map that maps a umi to the duplicateSetID
+
+    /**
+     * Create a map that maps a umi to the duplicateSetID
+     * @return
```

Comment:
If a javadoc tag is blank, better to omit it. In this case I would move the overall document into the return tag, e.g.
```
/**
 * @return a map that maps a UMI to the duplicateSetID
 */
```